exercises/practice/knapsack/.meta/Sources/Knapsack/Item.swift

```diff
@@ -0,0 +1,7 @@
+// Represents an item in knapsack.
+// There is be no need to edit this file and changes to this file are ignored by the Exercism test runners.
+
+struct Item {
```

Comment:
The indent should be 2 not 4 spaces. Otherwise lgtm

Reply:
I've just updated the exercise source files to use 2 spaces for indents. I wasn't sure about changing the indenting in the test template though. The test templates in the other exercises seem to use 4 spaces, although the formatter fixes the generated test to use 2 spaces. Let me know if the test template should also use 2 spaces.

Reply:
I'm not part of the Swift team but it sounds like the templates might benefit from an update. If they do get an update, that should probably be a different PR.

Reply:
The track was a bit of a mess when I took over it tbh, the concepts used 2 indent while the practice exercises used 4 (the online editor used 4). Swift doesn't seem to agree on the indent size. The Swift book (written by apple) uses 4 indent, but if you look at libraries apple writes they use 2 (an example for reference [here](https://github.com/swiftlang/swift-testing/blob/main/Sources/Testing/Traits/Bug.swift)), I also belive the formatted by default uses 2. Thereby there could have been a transition error when I decided to transition the track to 2 indent. You are free to update the templates files but I don't see it as a necessity

Reply:
Or actually, Swift 6 includes a new testing framework so that means that we should likely transition to that at some point which means re-writting all test files so likely the easiest thing would be to just apply the fix then.